managed/src/main/java/com/yugabyte/yw/models/helpers/CommonUtils.java

```diff
@@ -64,8 +64,8 @@ public static JsonNode maskConfig(JsonNode config) {
   }
 
   private static String getMaskedValue(String key, String value) {
-    return isStrictlySensitiveField(key) || (value == null) ? MASKED_FIELD_VALUE
-        : value.replaceAll(maskRegex, "*");
+    return isStrictlySensitiveField(key) || value.length() < 5
+        || (value == null) ? MASKED_FIELD_VALUE : value.replaceAll(maskRegex, "*");
```

Comment:
No-no-no. If you have this in such variant then you don't have a test case for this. IIRC, boolean operations are executed from left to right order. So if you have `value=null`, you will get NPE here as `value.length() < 5` is executed before `(value == null)`.
Update here and update tests to control this case.

Reply:
Oops!! Yeah, did not check the last condition. Will update and add a test case too.